client/server/bundler/utils.js

```diff
@@ -36,7 +41,18 @@ function getUrl( filename, hash ) {
 	);
 }
 
+const activeSections = config( 'sections' );
```

Comment:
How is this new `sections` config going to work with the existing section limit environment var? Regardless, we should make sure to document how it works.

Reply:
I think it works as expected. Included section (coming from the enviroment var) would need to be a subset of the active sections (the ones in the config) they get filtered out as expected. Right now there is only one but I can see it working if we had more. Since the filtering happens after we filter out the active sections.